src/mono/netcore/sample/Android/AndroidSampleApp.csproj

```diff
@@ -22,7 +22,7 @@
 
   <UsingTask TaskName="AndroidAppBuilderTask" AssemblyFile="$(AndroidAppBuilderTasksAssemblyPath)"/>
 
-  <Target Name="BuildApp" AfterTargets="CopyFilesToPublishDirectory">
+  <Target Name="BuildApp" AfterTargets="Build">
```

Comment:
same feedback about the Publish->Build change as I left in the iOS PR :)

Reply:
Reverted to the previous variant; in case of library test build, it will also depend on Publish.